query/sources/impl/src/test/groovy/org/codice/ddf/admin/sources/test/SourceCommonsSpec.groovy

```diff
@@ -124,78 +122,12 @@ class SourceCommonsSpec extends Specification {
             (USERNAME)       : TEST_USERNAME
     ]
 
-    static class TestRequestUtils extends RequestUtils {
```

Comment:
✏️ Why were TestRequestUtils and the code to mock the client moved out of this common class? It looks like all of the DiscoverSourcSpecsdo the same thing?

Reply:
I moved this to help increase readability within each test class. I'd also like to ultimately get rid of the SourceCommonsSpec class and not extend it anymore.